packages/shorebird_code_push_protocol/lib/src/messages/create_user/create_user_request.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+part 'create_user_request.g.dart';
+
+/// {@template create_user_request}
+/// The request body for POST /api/v1/users, which creates a new User.
+///
+/// Email is retrieved from the user's auth token.
```

Comment:
Seems a bit weird to require an access token for user creation imo. I would have expected the endpoint would be open and we'd provide email and display name

Reply:
Yeah I see makes sense to do that for now since that's simpler/safer. We can change it in the future 👍